src/content/docs/dropins/checkout/checkout-functions.mdx

```diff
@@ -1,9 +1,563 @@
+
+An array pf `RegionModel` objects is returned:
+
+```ts
+type Region = {
+  code: string;
+  id?: number;
+  name: string;
+};
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { getRegions } from '@/checkout/api/getRegions';
+
+getRegions();
+```
+
+# getStoreConfig
+
+Fetches information about a store's configuration settings from the Adobe Commerce GraphQL API using the [`storeConfig`](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/store-config/) query.
+
+You can query a non-default store by changing the header in your GraphQL request.
+
+### Returns
+
+The function returns a promise that resolves to the transformed store configuration data fetched from the API.
+
+If the API call fails, it returns the default store configuration settings (`STORE_CONFIG_DEFAULTS`).
+
+```ts
+transformStoreConfig
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { storeConfig } from '@/checkout/api/store/storeConfig';
+
+storeConfig();
+```
+
+## initializeCheckout
+
+Initializes the checkout process by fetching necessary configuration and data from various Adobe Commerce GraphQL APIs using the following functions:
+
+- `fetchAddressFormFields`
+- `getCountries`
+- `getStoreConfig`
+
+### Usage
+
+See the following example for usage details:
+
+```typescript
+import {
+  fetchAddressFormFields,
+  getCountries,
+  getStoreConfig,
+} from '@/checkout/api';
+
+initializeCheckout();
+```
+
+### Returns
+
+The function returns a promise that resolves to an array containing the results of the three API calls.
+
+```ts
+Promise.all([getStoreConfig(), getCountries(), fetchAddressFormFields()]);
+```
+
+## isEmailAvailable
+
+Checks whether the specified email has already been used to create a customer account using the [`isEmailAvailable`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/is-email-available/) Adobe Commerce GraphQL API query.
+
+A value of `true` indicates that the email address is available and the customer can use the email address to create an account.
+
+### Returns
+
+A promise that resolves to an `EmailAvailability` object indicating whether the email is available.
+
+```ts
+type EmailAvailability = boolean;
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { isEmailAvailable } from '@/checkout/api/isEmailAvailable';
+
+isEmailAvailable('test@example.com');
+```
+
+## placeOrder
+
+Handles the process of placing an order in the checkout system using the [`placeOrder`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/place-order/) Adobe Commerce GraphQL API mutation.
+
+### Returns
+
+The transformed order data fetched from the API.
+
+```ts
+transformOrder
+```
+
+### Usage
+
+See the following example for usage details:
+
+```ts
+import { placeOrder } from '@/checkout/api/placeOrder';
+
+placeOrder('4L7E1H0muABuggqWxbzBCBWoeG0rNxDh');
```

Comment:
`placeOrder()` API is without params (I guess mdx file was not updated correctly).